lib/mumukit/core/class.rb

```diff
@@ -0,0 +1,20 @@
+class Class
+  class ContractBreachError < Exception
+  end
+
+  class AbstractClassInstantiationError < Exception
+  end
+
+  unless %w(RACK_ENV RAILS_ENV).any? { |it| ENV[it] == 'production' }
+    alias_method :__new__, :new
```

Comment:
O think that defining a `__new__`  method may be risky since it is a common name pattern  - _double underscore_ - over a common name - `new`. 

I have found ~800 lines of code in github with ruby code that reference `__new__`: 

https://github.com/search?l=Ruby&q=__new__&type=Code

I'd rename it to something like  `__mumukit_core_contract_new__`

Reply:
sounds good!